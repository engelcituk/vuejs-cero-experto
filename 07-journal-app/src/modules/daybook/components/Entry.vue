<template>
    <div class="entry-container mb-3 pointer p-2"
        @click="$router.push({name:'entry', params:{id: 10}})"
    >
        <!-- title -->
        <div class="entry-title d-flex">
            <span class="text-success fs-5 fw-bold">08</span>
            <span class="mx-1 fs-5 fw-light">Septiembre </span>
            <span class="mx-2 fw-light">2021, Miercoles</span>
        </div>

        <div class="entry-description">
            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Similique, mollitia officiis. Quisquam itaque error vel! Saepe tempora ut sed officiis natus! Ea, autem! Sequi dicta cumque perferendis, recusandae dolore alias.
        </div>
    </div>
</template>
<style lang="scss" scoped>
    .entry-container {
        border-bottom: 1px solid #2c3e50;
        transition: 0.2s all ease-in;

        &:hover{
            background-color:lighten($color: grey, $amount: 45);
            transition: 0.2s all ease-in;
        }
        .entry-description{
            font-size: 12px;
        }
    }
</style>
