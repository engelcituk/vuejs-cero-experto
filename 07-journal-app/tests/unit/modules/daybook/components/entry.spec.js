
import { shallowMount } from '@vue/test-utils'
import Entry from '@/modules/daybook/components/Entry'

describe('Pruebas en el Entry component', ()=> {
    
    test('Debe de hacer match con el snapshot', ()=> {
        const wrapper = shallowMount( Entry )
        
    })   
      
    test('Debe de redireccionar al hacer click en el entry-container', ()=> {
        const wrapper = shallowMount( Entry )
        
    }) 

    test('Pruebas en las propiedades computadas', ()=> {
        const wrapper = shallowMount( Entry )
        
    }) 
})