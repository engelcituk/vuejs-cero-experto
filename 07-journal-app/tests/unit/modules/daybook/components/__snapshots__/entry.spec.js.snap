// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el Entry component Debe de hacer match con el snapshot 1`] = `
<div class="entry-container mb-3 pointer p-2">
  <!-- title -->
  <div class="entry-title d-flex"><span class="text-success fs-5 fw-bold">11</span><span class="mx-1 fs-5 fw-light">Septiembre</span><span class="mx-2 fw-light">2021, SÃ¡bado</span></div>
  <div class="entry-description">Texto de prueba</div>
</div>
`;
